src/nvim/ui_client.c

```diff
@@ -61,12 +61,17 @@ uint64_t ui_client_start_server(int argc, char **argv)
 
   Channel *channel = channel_job_start(args, get_vim_var_str(VV_PROGPATH),
                                        CALLBACK_READER_INIT, on_err, CALLBACK_NONE,
-                                       false, true, true, false, kChannelStdinPipe,
+                                       false, true, true, true, kChannelStdinPipe,
```

Comment:
This works just fine. Libuv is valuable here--saves a lot of headache trying to setup a "double fork" correctly.